packages/ui/lib/styled/Pagination.js

```diff
@@ -77,22 +81,38 @@ const Ellipsis = styled.div`
   }
 `;
 
+const noop = () => {};
+
+const defaultItemRender = (_page, _type, element) => {
+  return element;
+};
+
 export default function Pagination({
   page,
   pageSize,
   total,
-  setPage = () => {},
+  setPage = noop,
   large = false,
+  itemRender = defaultItemRender,
+  onChange = noop,
 }) {
   const totalPages = Math.ceil(total / pageSize)
     ? Math.ceil(total / pageSize)
     : 1;
   const PageItem = large ? LargeItem : Item;
 
+  const handleChange = (page) => {
+    setPage(page);
+    onChange(page);
```

Comment:
can we also pass `pageSize` to `onChange` ?
might be useful on scenarios

Reply:
`onChange(size, pageSize)` ?

Reply:
> `onChange(size, pageSize)` ?



> `onChange(size, pageSize)` ?

exactly